.travis.yml

```diff
@@ -4,4 +4,4 @@ rvm:
 before_script:
   - gem install awesome_bot
 script:
-  - awesome_bot README.md --allow-redirect --allow-dupe -w weibo.com,viosey.com,hackernoon.com
+  - awesome_bot README.md --allow-redirect --allow-dupe -w weibo.com,viosey.com,hackernoon.com,ghost.org
```

Comment:
No need to add `ghost.org` domain here. These domains for `w` option.

Reply:
So, you just did rebase or merge with master :)

Reply:
Reverted @YoshinoriN 👍